AcadeMeData/test_degree.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from AcadeMeData.models import Degree
+
+
+@pytest.mark.django_db
+class TestDegreeModel:
+    def degree_example(self):
```

Comment:
Please use fixture here
please refer the link below:

Consider open an Issue to make all fixtures in one file for laater work

https://github.com/redhat-beyond/AcadeMe/pull/66/commits/eb91a64b9f84f5a2a58202e63a39680c3f153490#diff-10c42fcf224bda15e60cdea601bebe0bbe71f0a4a5930d09dfac6243707b9933R95

Reply:
Done